sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/simple/SafeAddTests.java

```diff
@@ -0,0 +1,79 @@
+package org.dbsp.sqlCompiler.compiler.sql.simple;
+import org.dbsp.sqlCompiler.compiler.sql.SqlIoTest;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class SafeAddTests extends SqlIoTest {
+    @Test
+    public void testSafeAdd() {
+        this.qs(
+                "SELECT SAFE_ADD(2, 3) as 'result';\n" +
+                " result \n" +
+                "--------\n" +
+                "5\n" +
+                "(1 row)\n" +
+                "\n" +
+                "SELECT SAFE_ADD(9223372036854775807, 2) as 'result';\n" +
+                " result \n" +
+                "--------\n" +
+                " \n" +
+                "(1 row)\n" +
+                "\n" +
+                "SELECT SAFE_ADD(-9223372036854775807, -2) as 'result';\n" +
+                " result \n" +
+                "--------\n" +
+                " \n" +
+                "(1 row)\n" +
+                "\n" +
+                "SELECT SAFE_ADD(9.9, 1.0) as 'result';\n" +
+                " result \n" +
+                "--------\n" +
+                "10.9\n" +
+                "(1 row)\n" +
+                "\n" +
+                "SELECT SAFE_ADD(9.9, 1) as 'result';\n" +
+                " result \n" +
+                "--------\n" +
+                "10.9\n" +
+                "(1 row)\n" +
+                "\n" +
+                "SELECT SAFE_ADD(CAST(125 AS TINYINT), CAST(2 AS TINYINT)) as 'result';\n" +
+                " result \n" +
+                "--------\n" +
+                " 127\n" +
+                "(1 row)\n" +
+                "\n"
+        );
+    }
+
+    @Test
+    @Ignore
```

Comment:
you should add a reason for ignore with the issue link once you have filed the issue